DQM/TrackingMonitorSource/python/TrackCollections2monitor_cff.py

```diff
@@ -180,13 +180,84 @@
 doEffFromHitPatternVsBX             ['highPurityPt1'] = cms.bool(True)
 doStopSource                        ['highPurityPt1'] = cms.bool(True)
 
+###### all tracks (no pt cut) associated to the PV
+###### association is dz<1mm 
+from CommonTools.RecoAlgos.TrackWithVertexSelector_cfi import *
+
+trackAssociated2pvSelector = trackWithVertexSelector.clone()
+trackAssociated2pvSelector.trackWithVertexSelectorParams = cms.PSet(
```

Comment:
`TrackWithVertexSelector` does not have(/read) `trackWithVertexSelectorParams` PSet.

What happens in `TrackWithVertexSelector_cfi.py` is that the contents of `trackWithVertexSelectorParams` are copied to `trackWithVertexSelector`.